.github/workflows/playwright.yml

```diff
@@ -66,7 +66,7 @@ jobs:
     if: ${{ github.event.pull_request.title == null || needs.build-plugin.outputs.changelog_diff }}
```

Comment:
Can you please create a new mindmap, specifically for this task?
The work technically is very straightforward, but we need to make sure that we don't overlook anything. A mindmap will be very helpful for that.

Reply:
Extended existing Mind Map with new some new staff related to this ticket 
https://whimsical.com/rating-internal-qa-task-check-the-widget-controls-and-behaviour--663DNaSgeGPw4epPdc6X6x

Reply:
@mykytamurzin I am curious, can you please describe how you have tested that the old widget is still working correctly? Just curious :).

Reply:
I did QA task for new rating and since then I have a page with an old star rating widget

Reply:
Nice.